.github/workflows/pr_checklist.yml

```diff
@@ -14,22 +14,28 @@ jobs:
         with:
           script: |
             const comment = `
-            Thanks for creating a pull request! The below questions and checklist are intended to help with verifying your change is well tested. Response is optional, but if you choose to respond please edit this comment.
+            Thanks for creating a pull request! The below questions and checklist are intended to help with verifying your change is well tested. Please edit this comment by answering the questions below, or replacing the questions with your own description. Please remove this paragraph once you finish editing this comment.
```

Comment:
Remove the "optional" wording to promote adoption. Also, add a suggestion to remove the 1st paragraph so that it's more visible/obvious to reviewers that the PR comment has been updated.